### PR TITLE
USAGOV-1969 Manage search result weights

### PR DESCRIPTION
**Pull request by omerida**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1969

## Description

Installs the sortableviews module to provide a drag-and-drop UI for managing the weight field of search results.

## Type of Changes

- [x] New Feature
- [ ] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [x] Drupal Config (requires "drush cim")
- [x] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] WWW
  - [ ] Egress
  - [ ] Tools
  - [ ] Cron
- [ ] Other

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [ ] Requires New Config
- [ ] Requires New Content

### Validation Steps

- On English benefit finder page, should see a local task linking to the sorting view. 
 
image

   - Clicking on the link should take you to the view for sorting English benefit pages. `/admin/manage-benefits-search-sort/en`

image

    - Change the order of some items (click on handle to activate , then click and drag to new location).
    - Return to the benefit search page and confirm your new order is reflected in the search results. May need a cache refresh to get the latest json file for search order.

- Spanish benefit finder page should have a similar local task that goes to `/admin/manage-benefits-search-sort/en` and affects the order of search results in the same way as above.
- Confirm the local task does not show up on other pages.
- Confirm the local task shows up if you're logged in as a Content Administrator or Site Admin
- Confirm the local task does not show up for other roles.

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: config/sync/system.logging.yml
Comment (akf) [inline config/sync/system.logging.yml:1]: Was this just for testing, by chance?

**Review by akf — CHANGES_REQUESTED** [review]
It looks good but I'd recommend we wait until after tomorrow's push to staging to merge it, so the content team (okay, Shoshana) can give it a thorough look. 

Also, I don't think we want verbose logging, unless I've missed something. (Even though the cms isn't meant to be for the public, it is exposed.)

**Comment by omerida** [comment]
yes, that was for testing, I kept unstaging it but I missed it in the end

File: config/sync/system.logging.yml
Comment (omerida) [reply config/sync/system.logging.yml:1]: Fixed. I kept getting a WSOD with the local tasks

**Review by akf — APPROVED** [review]

**Pull request merged**
